src/libs/ReportActionsUtils.js

```diff
@@ -134,7 +134,7 @@ function isConsecutiveActionMadeByPreviousActor(reportActions, actionIndex) {
 function getLastVisibleAction(reportID, actionsToMerge = {}) {
     const actions = _.toArray(lodashMerge({}, allReportActions[reportID], actionsToMerge));
     const visibleActions = _.filter(actions, action => (!isDeletedAction(action)));
-    return _.max(visibleActions, action => moment.utc(action.created).valueOf());
+    return _.max(visibleActions, action => moment.utc(action.created).valueOf()) || {};
```

Comment:
This won't work. `-Infinity || {}` will always return `-Infinity`.
You should add an early return instead if the visible actions array length is zero then return empty object.

Reply:
Oh, thanks! I had some misunderstanding between `underscore` and `lodash` library, their `max` functions are different. `underscore`'s returns `-Infinity` while the other returns `undefined`.